src/sections/disease/Ontology/OntologySubgraph.js

```diff
@@ -289,39 +299,52 @@ function OntologySubgraph({
           <g transform={`translate(0, ${yOffset})`}>
             {nodes.map(node => {
               return (
-                <Fragment key={node.id}>
-                  <text
-                    x={node.y - xOffset}
-                    y={node.x}
-                    dx="9"
-                    fontSize="12"
-                    dominantBaseline="middle"
-                    fill="#5a5f5f"
+                <Link
+                  to={`/disease/${node.data.id}`}
+                  className={classNames(classes.labelText)}
```

Comment:
Since it's only one class, we could write this as `className={classes.labelText}`